modyn/supervisor/internal/grpc/supervisor_grpc_server.py

```diff
@@ -18,7 +19,7 @@ def callback(modyn_config: dict, server: Any, supervisor: Supervisor) -> None:
 
     def __init__(self, modyn_config: dict) -> None:
         self.modyn_config = modyn_config
-        self.supervisor = Supervisor(modyn_config)
+        self.supervisor = Supervisor(ModynConfig(**modyn_config))
```

Comment:
Internally we want to use the pydantic model, however externally we cannot easily offer the ModynConfig as an argument as many tests depend on this. Refactoring the tests in this regard isn't worth it.

Reply:
Sounds good, let's keep the cost-benefit tradeoff here :) Thank you so much